src/components/addSourceWizard/hardcodedComponents/azure/costManagement.js

```diff
@@ -202,10 +213,280 @@ export const ExportSchedule = () => {
         <TextListItem component={TextListItemVariants.dd}>
           {intl.formatMessage({
             id: 'cost.azure.createdAccountName',
-            defaultMessage: 'Storage account name created earlier',
+            defaultMessage: 'Created storage account name or existing storage account name',
+          })}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          <Text component={TextVariants.b}>
+            {intl.formatMessage({
+              id: 'cost.azure.resourceGroupName',
+              defaultMessage: 'Resource group name',
+            })}
+          </Text>
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {intl.formatMessage({
+            id: 'cost.azure.createdResourceGroupName',
+            defaultMessage: 'Resource group for the storage account',
           })}
         </TextListItem>
       </TextList>
+      <br />
+      <Text component={TextVariants.p}>
+        {intl.formatMessage({
+          id: 'cost.azure.exportNameFollowup',
+          defaultMessage: 'After configuring the daily export, enter the following:',
+        })}
+      </Text>
     </TextContent>
   );
 };
+
+class CostExportSelect extends React.Component {
```

Comment:
Can we use functional component instead of a class?

Reply:
Yep. that's what you get from a backend dev 😆